tests/cases/docker/vscode/Dockerfile

```diff
@@ -10,6 +10,7 @@ COPY --from=typescript/typescript /typescript/typescript-*.tgz /typescript.tgz
 WORKDIR /vscode/build
 RUN yarn add typescript@/typescript.tgz
 WORKDIR /vscode/extensions
+RUN yarn add rimraf

```

Comment:
Is this working around a bug in vscode's own build? Since we clearly don't sue rimraf here in the dockerfile.

Reply:
Yes. The `vscode` build is failing because they have a postinstall.js script that runs rimraf to clean up TypeScript files they don't need, but we're running `yarn add` for our copy of TypeScript *before* all the dependencies are installed which triggers the postinstall script.

Reply:
Does yarn have a `--ignore-scripts` equivalent we should use instead?

Reply:
Nope, not according to the docs for `yarn add`.